lib/watir/locators/element/selector_builder/xpath.rb

```diff
@@ -49,7 +49,8 @@ def attribute_expression(building, selectors)
           def equal_pair(building, key, value)
             if key == :class
               if value.strip.include?(' ')
-                Watir.logger.deprecate "Using the :class locator to locate multiple classes with a String value", "Array"
+                Watir.logger.deprecate "Using the :class locator to locate multiple classes with a String value (i.e. #{value})",
+                                       "Array (e.g. [#{value.split}])"
```

Comment:
This still doesn't quite format correctly. A value of "a b" will result in:
* "Using the :class locator to locate multiple classes with a String value (i.e. a b)" - should include quotes around value?
* "Array (e.g. [[\"a\", \"b\"]])" - duplicates "[" and "]"

Reply:
oops, yes, I left in the brackets, should just be:
```
"Array (e.g. #{value.split})"
```

Reply:
Did you want to quote the deprecated String value - ie `(i.e. "a b")` instead of `(i.e. a b)`?

Reply:
yes, and I'll even fix the associated spec while I'm at it